modules/2_classification/test/test_classifier.cpp

```diff
@@ -80,7 +80,10 @@ TEST(classification, DenseNet) {
         classesNames.push_back(line);
 
     // TODO: add code here
-
+	Classifier cl;
+	int k = 5;
+	std::vector<int> indices;
+	cl.classify(image, k, probabilities,indices);
```

Comment:
Please do the following:
1. remove comment "TODO"
2. Fix indentation
3. Use `k` in the checks below:
```cpp
    ASSERT_EQ(top5_scores.size(), k);
    ASSERT_EQ(top5_classes.size(), k);
```